_episodes/04-building-containers.md

```diff
@@ -144,6 +147,8 @@ We will automate this in the next section.
 > > ...
 > > ~~~
 > > {: .output}
-> > Notice how we did not need `--fakeroot` for the installation since pip installs user packages.
+> > Notice how we did not need neither `--writable ` nor `--fakeroot` for the installation, but everything worked fine since pip installs user packages, in the user $HOME directory.
+> > And Singularity/Apptainer by default mounts as read+write the user home directory, even if the container is read-only.
```

Comment:
"In addition, Singularity/Apptainer by default mounts the user home directory as read+write, even if the container is read-only."

Reply:
included, thanks